Building-with-IntelliJ-IDEA.md

```diff
@@ -20,29 +20,39 @@ You can build RuneLite locally using [JDK 11](https://adoptium.net/temurin/relea
 
 ## Importing the project
 
-After launching IntelliJ IDEA for first time, you will see IntelliJ welcome window. You will now need to clone RuneLite repository from git:
+You will now need to clone RuneLite repository from git. After launching IntelliJ IDEA for first time, you will see IntelliJ welcome window. click on `Get from VCS`.
 
-![welcome-import-git](https://i.imgur.com/bALnvDY.png)
+![get_from_vcs.png](img%2Fbuilding-with-intellij%2Fget_from_vcs.png)
+
+If you're presented with the editor instead of the welcome screen, click on on  `File > New > Project from Version Control…`
```

Comment:
Is there a reason why this is a better instruction then telling them to close their old project? They'll end up either having to close it later or have 2 projects open.

If anything a normal user following this build guide would not have this issue, unless they previously ran some project and forgot about it, which instructions to tell them to close from the beginning, seem cleaner.

Reply:
The extra step is not avoided with this approach tho, you are still prompted later-on whether or not you'd like to load your new project in the same window or a new one

Reply:
Letting the user choose whether they want to open the project in a new window or not should be their own decision and not necessary to the instructions.

Reply:
If you _want_ to have multiple windows open, you already have some form of familiarity with IntelliJ and don't need to read these instructions. It seems much easier to a new user to be given direct instructions to close the existing project, so they don't have a random prompt sprung on them later, that isn't mentioned.